packages/web-client/src/helpers/resource/types.ts

```diff
@@ -51,6 +51,7 @@ export interface Resource {
   thumbnail?: string
   status?: number
   processing?: boolean
+  locked?: boolean
```

Comment:
not needed in this PR but i use this property to set the locked status of a file